pixi/config.js

```diff
@@ -1,11 +1,13 @@
 module.exports = {
   development: {
+    API_ENDPOINT_LINTER: 'http://localhost:8080/page-experience/api/lint/?url=',
```

Comment:
Can we use a Links without the server part here?
We could also use it the same way for the production linter setting then and it would work for the staging environment as well.

Reply:
Sadly no, amp-script requires fetched URLs to be absolute.